sql/migrations/20210515110157_world.sql

```diff
@@ -0,0 +1,851 @@
+(3222,70,-2227.31,-420.418,-9.3886,100,0,0),
+(3222,71,-2211.36,-400.907,-8.07511,100,0,0),
+(3222,72,-2196.92,-395.961,-5.9352,100,0,0),
+(3222,73,-2178.88,-392.144,-3.76367,100,0,0),
+(3222,74,-2194.74,-391.121,-5.85785,100,0,0),
+(3222,75,-2219.08,-386.393,-9.35368,100,0,0),
+(3222,76,-2236.59,-385.575,-9.42354,100,0,0),
+(3222,77,-2248.19,-387.537,-9.42354,100,0,0),
+(3222,78,-2264.4,-389.364,-9.42354,100,0,0),
+(3222,79,-2273.3,-383.892,-9.42354,100,0,0),
+(3222,80,-2285.88,-375.369,-9.4112,100,0,0),
+(3222,81,-2294.18,-374.99,-9.29083,100,0,0),
+(3222,82,-2304.23,-376.537,-9.22788,100,0,0),
+(3222,83,-2310.61,-387.123,-9.10346,100,0,0),
+(3222,84,-2315.59,-393.766,-8.63748,100,0,0),
+(3222,85,-2323.53,-396.452,-8.12964,100,0,0),
+(3222,86,-2329.52,-396.651,-8.07738,100,0,0),
+(3222,87,-2351.43,-393.596,-8.11827,100,0,0);
+
+-- Brave Cloudmane 3224 (credit cmangos)
+DELETE FROM `creature_movement` WHERE `id` IN (26053);
+DELETE FROM `creature_movement_template` WHERE `entry` = 3224;
+INSERT INTO `creature_movement_template` (`entry`, `point`, `position_x`, `position_y`, `position_z`, `orientation`, `waittime`, `wander_distance`, `script_id`) VALUES
+(3224,1,-2260.23,-299.509,-9.34905,100,0,0,0),
+(3224,2,-2261.29,-302.85,-9.29993,100,0,0,0),
+(3224,3,-2260.51,-306.049,-9.29993,100,0,0,0),
+(3224,4,-2257.75,-312.749,-9.29993,100,5000,5,0),
+(3224,5,-2265.13,-312.055,-9.42604,100,0,0,0),
+(3224,6,-2272.79,-315.722,-9.42604,100,5,5000,0),
+(3224,7,-2270.73,-308.69,-9.42604,100,0,0,0),
+(3224,8,-2274.7,-300.973,-9.42604,100,5000,5,0),
+(3224,9,-2270.95,-301.636,-9.29993,100,0,0,0),
+(3224,10,-2267.42,-302.304,-9.29993,100,0,0,0),
+(3224,11,-2263.55,-300.676,-9.29993,100,0,0,0),
+(3224,12,-2259.32,-296.491,-9.34905,100,5000,5,0);
+
+-- Explosive Sheep 2675 (credit cmangos)
+INSERT INTO `creature_ai_events` (`id`, `creature_id`, `condition_id`, `event_type`, `event_inverse_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action1_script`, `action2_script`, `action3_script`, `comment`) VALUES (267501, 2675, 0, 11, 0, 100, 0, 0, 0, 0, 0, 267501, 0, 0, 'Explosive Sheep - Cast Explosive Sheep Passive and Stealth Detection on Spawn');
+INSERT INTO `creature_ai_scripts` (`id`, `delay`, `command`, `datalong`, `datalong2`, `datalong3`, `datalong4`, `target_param1`, `target_param2`, `target_type`, `data_flags`, `dataint`, `dataint2`, `dataint3`, `dataint4`, `x`, `y`, `z`, `o`, `condition_id`, `comments`) VALUES (267501, 0, 15, 4051, 0, 0, 0, 0, 0, 0, 4, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Explosive Sheep - Cast Spell');
+INSERT INTO `creature_ai_scripts` (`id`, `delay`, `command`, `datalong`, `datalong2`, `datalong3`, `datalong4`, `target_param1`, `target_param2`, `target_type`, `data_flags`, `dataint`, `dataint2`, `dataint3`, `dataint4`, `x`, `y`, `z`, `o`, `condition_id`, `comments`) VALUES (267501, 0, 15, 8279, 0, 0, 0, 0, 0, 0, 4, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Explosive Sheep - Cast Spell');

```

Comment:
Passive auras can be placed in the `auras` field in `creature_template`. I don't think you need EventAI for this.